administration/src/bp-modules/statistics/StatisticsOverview.tsx

```diff
@@ -1,23 +1,47 @@
 import React, { ReactElement, useContext } from 'react'
 
 import { WhoAmIContext } from '../../WhoAmIProvider'
-import { CardStatisticsResultModel, Role } from '../../generated/graphql'
+import { CardStatisticsResultModel, Region, Role } from '../../generated/graphql'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import downloadDataUri from '../../util/downloadDataUri'
+import { useAppToaster } from '../AppToaster'
+import { CsvStatisticsError, generateCsv, getCsvFileName } from './CSVStatistics'
 import StatisticsBarChart from './components/StatisticsBarChart'
 import StatisticsFilterBar from './components/StatisticsFilterBar'
 import StatisticsTotalCardsCount from './components/StatisticsTotalCardsCount'
 
 type StatisticsOverviewProps = {
   statistics: CardStatisticsResultModel[]
   onApplyFilter: (dateStart: string, dateEnd: string) => void
+  region?: Region
 }
 
-const StatisticsOverview = ({ statistics, onApplyFilter }: StatisticsOverviewProps): ReactElement => {
+const StatisticsOverview = ({ statistics, onApplyFilter, region }: StatisticsOverviewProps): ReactElement => {
   const { role } = useContext(WhoAmIContext).me!
+  const appToaster = useAppToaster()
+  const { cardStatistics } = useContext(ProjectConfigContext)
+  const exportCardDataToCsv = (dateStart: string, dateEnd: string) => {
+    try {
+      downloadDataUri(generateCsv(statistics, cardStatistics), getCsvFileName(`${dateStart}_${dateEnd}`, region))
+    } catch (error) {
+      if (error instanceof CsvStatisticsError) {
```

Comment:
Maybe I'm missing something, but is this condition necessary?
Currently I don't see if other types of errors can occur here, but in general, why not just show a notification in case of any error?

Reply:
yes you are right.